### PR TITLE
0.2.0 release

### PR DESCRIPTION
**Pull request by TomasTomecek**

### CODE REVIEW

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: success, return code = __0__ 
* Result url: __[https://da.gd/IQIM](https://da.gd/IQIM)__
* Comment: __See details for logs__

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: success, return code = __0__ 
* Result url: __[https://da.gd/JQe90](https://da.gd/JQe90)__
* Comment: __See details for logs__

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+* We added support for docker-py version 1 so it can work in CentOS.
+* conu will check whether required binaries are present and if not,
+  `CommandDoesNotExistException` will be raised.
+* Our documentation was improved and contains now more examples and the python
+  interface should be explained in more detail.
+* When creating docker containers, conu now utilizes option `--cidfile`.
+* `execute()` method can be blocking and non-blocking.
+* Backend provides a new methos `cleanup_containers()` to cleanup containers
+  and images created during the session.
```
Comment (dhodovsk) [inline CHANGELOG.md]: It can now cleanup only containers.

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: success, return code = __0__ 
* Result url: __[https://da.gd/eTntE](https://da.gd/eTntE)__
* Comment: __See details for logs__

**Pull request merged**